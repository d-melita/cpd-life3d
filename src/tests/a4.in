10 512 .4 0
