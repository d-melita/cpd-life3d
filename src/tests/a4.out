1 19157196 9
2 11835190 9
3 10389985 1
4 9659849 1
5 9049681 1
6 8563492 1
7 8146694 1
8 7824521 1
9 7580102 1
